### PR TITLE
feature/web3-507-peaq-platform-extract-as-standalone-app-and-make-it-work-part-2

### PR DESCRIPTION
**Pull request by grmkris**
**JIRA task url** (if applicable)

**(If Bug) What was the problem being solved?**

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

**How to reproduce fix / test this PR?**

### CODE REVIEW

File: v2/airdrop-platform/app/src/configureDemoEnv.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { env } from "@/env.mjs";
+import { _setInternalConfig as _setInternalConfigReact } from "@nexeraid/react-sdk";
+import { _setInternalConfig as _setInternalConfigNode } from "@nexeraid/js-sdk";
+
+console.log("_setInternalConfigReact", { env: env.NEXT_PUBLIC_ENVIRONMENT });
```
Comment (prevostc) [inline v2/airdrop-platform/app/src/configureDemoEnv.tsx:5]: we should probably add a comment here that this is not needed
or we add a todo to remember to remove that when workflows are in prod

File: v2/airdrop-platform/app/src/kyc-airdrop/config/CONTRACT_ADDRESSES.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import type { Address } from "viem";
+
+// Contract addresses for different environments and chains
+export const CONTRACT_ADDRESSES: {
+  [key in Environment]: Partial<{
+    [key in EvmChainId]: { token: Address; distributor: Address }
+  }>
+} = {
+  prod: {
+    [NEXERA_EVM_CHAINS.SEPOLIA]: {
```
Comment (prevostc) [inline v2/airdrop-platform/app/src/kyc-airdrop/config/CONTRACT_ADDRESSES.ts:12]: that's something that should be in the SDK tbh

File: v2/airdrop-platform/app/src/kyc-airdrop/config/CUSTOMERS_BALANCE_MAP.ts [hunk 3]
```diff
@@ -0,0 +1,539 @@
+import { type BalanceMap } from "@nexeraid/merkle-tree-js";
+
+export const CUSTOMERS_BALANCE_MAP: BalanceMap[] = [
```
Comment (prevostc) [inline v2/airdrop-platform/app/src/kyc-airdrop/config/CUSTOMERS_BALANCE_MAP.ts:3]: if that's generated from CSV, should probably add a comment here explaining that?

**Review by prevostc — APPROVED** [review]

File: v2/airdrop-platform/app/src/kyc-airdrop/config/CONTRACT_ADDRESSES.ts (hunk 2 above)
Comment (joelamouche) [reply v2/airdrop-platform/app/src/kyc-airdrop/config/CONTRACT_ADDRESSES.ts:12]: WHich SDK? These are example addresses. I would rename this EXAMPLE_AIRDROP_CONTRACT_ADDRESSES

Comment (prevostc) [reply v2/airdrop-platform/app/src/kyc-airdrop/config/CONTRACT_ADDRESSES.ts:12]: Ah, mb then, agreed on the rename

**Pull request merged**